internal/vegeta/report.go

```diff
@@ -120,5 +121,36 @@ decode:
 		return "", err
 	}
 
```

Comment:
[Add comment]
// Adapt vegeta status code (type: map) to list of type: statusCode{}, as defined in swagger spec